connector-packager/tests/test_resources/oauth_connector/connectionFields.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should attempt to make this a roughly conforming oauth connector:
- an attrAthentication field
- either user/pass and oauth, or just oauth only (easiest): we have references if you need.
- correct required-attributes

Reply:
👍